storage.bs

```diff
@@ -239,6 +247,9 @@ steps:
  <li><p>If <var>A</var>'s <a for="storage key">origin</a> is not <a>same origin</a> with
  <var>B</var>'s <a for="storage key">origin</a>, then return false.
 
+ <li><p>If <var>A</var>'s <a for="storage key">top-level site</a> is not equal to
```

Comment:
Oh interesting. I guess this is the first time we have to define "same site" for two sites. Shall we add that to HTML?

Reply:
Yeah I think we should, unless we're willing to add general tuple-equality to Infra.

Reply:
Or I could serialize and use string comparison?

Reply:
Took a stab at a site same site PR in: https://github.com/whatwg/html/pull/7987

Reply:
I'm not sure how to reference the data-x thing here in the storage spec, though.

Reply:
You can use `<a for=site>same site</a>` it seems. It won't work until the HTML PR has landed and is indexed.